src/main/java/io/turntabl/employementprofilingsystem/Controllers/AuthController.java

```diff
@@ -0,0 +1,66 @@
+package io.turntabl.employementprofilingsystem.Controllers;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.turntabl.employementprofilingsystem.Transfers.*;
+import io.turntabl.employementprofilingsystem.Utilities.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataAccessException;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.web.bind.annotation.*;
+
+import java.sql.SQLException;
+import java.util.*;
+
+@Api
+@RestController
+public class AuthController {
+    @Autowired
+    JdbcTemplate jdbcTemplate;
+
+    Parsor parsor = new Parsor();
+
+    @ApiOperation("Get Employee By Email")
+    @CrossOrigin(origins = "*")
+    @GetMapping("/v1/api/login/{employee_email}")
+    public Map<String,Object> getEmployeeByEmail(@PathVariable("employee_email") String email) {
+
+        Map<String, Object> request = new HashMap<>();
+        Map<String, Object> response = new HashMap<>();
+
+        try{
+            request.put("employee_email", email);
+
+            List<String> requiredParams = Arrays.asList(
+                    "employee_email"
+            );
+            Map<String, Object> valid = parsor.validate_params(request, requiredParams);
+            if (valid.get("code").equals("00")) {
+                List<Employee>  employee = jdbcTemplate.query(
+                        "select * from employee where employee_email = ?",
```

Comment:
don't select * - what if someone adds a column (which is exactly what you're doing!) - all the existing queries will break. Instead, make your code more resilient by naming the columns in the select clause.